### PR TITLE
Fix i18n tags for Generate Image/Stop Image Generation, add translations

### PR DESCRIPTION
**Pull request by ceruleandeep**
Will ask @RivelleDays to have a look, this is 1 of 3 small PRs related to i18n 

## Checklist:

- [X] I have read the [Contributing guidelines](https://github.com/SillyTavern/SillyTavern/blob/release/CONTRIBUTING.md).

### CODE REVIEW

**Comment by RivelleDays** [comment]
I've checked the Simplified and Traditional Chinese parts, and everything looks good so far! If there are any issues or if further refinements are needed later, I'll submit another PR.

File: public/locales/zh-tw.json [hunk 1]
```diff
@@ -1446,107 +1447,107 @@
     "(-1 for random)": "（-1 表示隨機）",
     "(Optional)": "（可選）",
     "(use _space": "（使用",
-    "[no_connection_text]api_no_connection": "未連線⋯",
```
Comment (Cohee1207) [inline public/locales/zh-tw.json:1449]: Pls return the attribute specifiers back.

Comment (ceruleandeep) [reply public/locales/zh-tw.json:1449]: The highlighted change fixes this, have I missed something else?

online_status_text_afore
online_status_text_after

Comment (ceruleandeep) [reply public/locales/zh-tw.json:1449]: reason for doing it was e.g. "[title]Duplicate persona": "複製使用者角色" gives untrans tooltip:

dupl-persona-zhtw-afore
dupl-persona-zhtw-after

in zh-CN it has always been "Duplicate persona": "复制用户角色" and has worked/does work:
dupl-persona-zhcn-afore
dupl-persona-zhcn-after

Comment (Cohee1207) [reply public/locales/zh-tw.json:1449]: Hm, yeah, it appears so.

**Pull request merged**